content/documentation/pages/1-installation/3-kubernetes/4-compatibility.md

```diff
@@ -18,9 +18,10 @@ release versions.
 The following listing outlines the compatibility between Spring Cloud
 Data Flow for Kubernetes Server and Kubernetes versions:
 
-|                                              | Kubernetes 1.11.x | Kubernetes 1.12.x | Kubernetes 1.13.x | Kubernetes 1.14.x | Kubernetes 1.16.x | Kubernetes 1.17.x |
-| -------------------------------------------- | :---------------: | :---------------: | :---------------: | :---------------: | :---------------: | :---------------: |
-| SCDF K8S Server: 2.5.x - K8S Deployer: 2.3.0 |        ❌         |        ❌         |        ✅         |        ✅         |        ✅         |        ✅         |
-| SCDF K8S Server: 2.4.x - K8S Deployer: 2.2.0 |        ❌         |        ❌         |        ✅         |        ✅         |        ❌         |        ❌         |
-| SCDF K8S Server: 2.3.x - K8S Deployer: 2.1.1 |        ✅         |        ✅         |        ✅         |        ❌         |        ❌         |        ❌         |
-| SCDF K8S Server: 2.2.x - K8S Deployer: 2.0.4 |        ✅         |        ✅         |        ✅         |        ❌         |        ❌         |        ❌         |
+|                                              | Kubernetes 1.11.x | Kubernetes 1.12.x | Kubernetes 1.13.x | Kubernetes 1.14.x | Kubernetes 1.15.x | Kubernetes 1.16.x | Kubernetes 1.17.x |
+| -------------------------------------------- | :---------------: | :---------------: | :---------------: | :---------------: | :---------------: | :---------------: | :---------------: |
+| SCDF K8S Server: 2.6.x - K8S Deployer: 2.4.0 |        ❌         |        ❌         |        ❌         |        ❌         |        ❌         |        ✅         |        ✅         |
```

Comment:
I'd switch all the K8s-deployer versions to 2.1.x, 2.2.x, 2.3.x, and so on. Let's make them all release-line specific.

Reply:
updated